### ISSUE TITLE
Standalone mode

### ISSUE THREAD

**Issue by Phyks** [konnectors/libs#176]
Hi,

I'm super interested in running some konnectors in a standalone mode to store the fetched data in my own data storage, and not in the whole Cozy stack. I came across the `yarn standalone` feature, which should do the trick.

However, it looks like this logs all the fetched data, but does not dump anything (I guess it could dump an `importedData.json` file for instance).

I may have missed something. Otherwise, would you accept a PR to:
1. Let the user pass fields in an environment variable and not necessarily in a predefined plaintext path?
2. Dump all the fetched data in `folderPath/importedData.json`?

Thanks!

**Comment by Phyks** [comment]
Thanks! I'll have a look at this then, and write a PR.

**Comment by Phyks** [comment]
@doubleface I started to work on this, and should have something to dump created documents into a JSON file. I tested it with Ameli connector and it should work fine. Let me know if you have any comments on this.

Once I have some feedback on this, I plan to work on the possibility to pass fields in an environment variable.

One small question also, why are some connectors forcing SENTRY tokens? Typically [this](https://github.com/konnectors/cozy-konnector-ameli/blob/master/src/index.js#L5-L7). Should I write a PR to let the user overcome the `SENTRY` tokens?
